manuscript.md

```diff
@@ -42,7 +42,15 @@
 
 ## How are interaction strengths actually inferred?
 
-There are many ways to infer interaction strength, whether it is empirically or theoretically. 
+**General discussion of interaction strength inference method**
+- Interaction strength can either be infered empirically or theoretically [@Berlow2004IntStr]. Theoretical inference can be more mechanistic or with the Lotka-Volterra framework.
```

Comment:
I'm thinking relating the beginning of this section to functional response and the early metabolic papers ([Yodzis and Innes 1992](https://www.journals.uchicago.edu/doi/abs/10.1086/285380), [Brown et al 2004](www.dx.doi.org/10.1890/03-9000), [Rooney and McCann's book](https://link.springer.com/chapter/10.1007/978-1-4020-5337-5_2)) could help